docs/how-tos/provision-setup.md

```diff
@@ -154,7 +154,7 @@ if __name__ == '__main__':
 If you are using the captive portal, this step is optional.
 If you are using a mobile app, you must create a provisioning configuration file, specifying at least a `fragment_id`.
 
-Create a file called <FILE>viam-provisioning.json</FILE> with the following format and customize the [attributes](/fleet/provision/#configuration):
+Create a file called <FILE>/etc/viam-provisioning.json</FILE> with the following format and customize the [attributes](/fleet/provision/#configuration):
```

Comment:
Is this change still needed? @abe-winter and @Otterverse? I think I tested this and the script asks you to enter where the json is anyway, doesn't it?

Reply:
guessing not -- would discard all my changes, I didn't understand the context for these instructions

Reply:
```suggestion
Create a file called <FILE>viam-provisioning.json</FILE> with the following format and customize the [attributes](/fleet/provision/#configuration):
```